packages/kwasm/v0.2.3+1/package.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "kwasm"
+shortDescription: "The KWasm Operator allows for fine-grained control over node provisioning instead of using the node installer with a DaemonSet"
```

Comment:
Can we add this suggested change, or something similar? Because IMO the original shortDescription isn't very telling about what the package actually does.